.github/actions/setup-docker-environment/action.yaml

```diff
@@ -32,14 +32,14 @@ runs:
         version: latest
 
     - name: Login to DockerHub
-      if: ${{ github.event_name == 'release' || github.event_name == 'push' && github.ref == 'refs/heads/master' }}
+      if: ${{ github.event_name == 'release' || github.event_name == 'push' && github.ref == 'refs/heads/master' && inputs.password != ''  }}
       uses: docker/login-action@v2
       with:
         username: ${{ inputs.username }}
         password: ${{ inputs.password }}
 
     - name: Login to GitHub Container Registry
-      if: ${{ github.event_name == 'release' || github.event_name == 'push' && github.ref == 'refs/heads/master' }}
+      if: ${{ github.event_name == 'release' || github.event_name == 'push' && github.ref == 'refs/heads/master' && inputs.password != ''  }}
```

Comment:
`inputs.password` is for DockerHub, we should check for `inputs.ghcr_password` when checking to see if we should log into the GitHub registry?

Reply:
Ah, good catch! Fixed in 9831dfe